content/docs/refs-and-the-dom.md

```diff
@@ -31,30 +31,37 @@ Your first inclination may be to use refs to "make things happen" in your app. I
 
 ### Adding a Ref to a DOM Element
 
+>**Note:**
+>
+>The examples below have updated to use the new `React.createRef()` API, introduced in React 16.3.
+
+
 React supports a special attribute that you can attach to any component. The `ref` attribute takes a callback function, and the callback will be executed immediately after the component is mounted or unmounted.
 
-When the `ref` attribute is used on an HTML element, the `ref` callback receives the underlying DOM element as its argument. For example, this code uses the `ref` callback to store a reference to a DOM node:
+When the `ref` attribute is used on an HTML element, the `ref` created in the constructor with `React.createRef()` receives the underlying DOM element as its `value` property. For example, this code uses a `ref` to store a reference to a DOM node:
 
 ```javascript{8,9,19}
 class CustomTextInput extends React.Component {
   constructor(props) {
     super(props);
+    // create a ref to store the textInput DOM element
+    this.textInput = React.createRef();
```

Comment:
These changes break the line highlighting above
> javascript{8,9,19}

You'll want to update the line number offsets.

Might also be a nice time to move these examples into the ["examples" folder](https://github.com/reactjs/reactjs.org/tree/master/examples) so Prettier can manage the formatting for us. (Totally optional though.) This would allow you to use the nicer "highlight-line" and "highlight-range" directives.

Reply:
Fixed. 👍